osf/models/analytics.py

```diff
@@ -108,9 +107,8 @@ def update_counter(cls, resource, file, version, action, node_info, session_key)
         cleaned_page = cls.clean_page(page)
         date = timezone.now()
         date_string = date.strftime('%Y/%m/%d')
-        session_obj = session_store.get(session_key=session_key)
-        session_data = session_obj.get_decoded()
-        visited_by_date = session_data.get('visited_by_date', {'date': date_string, 'pages': []})
+        session_obj = SessionStore(session_key=session_key)
+        visited_by_date = session_obj.get('visited_by_date', {'date': date_string, 'pages': []})
```

Comment:
hmm, we no longer need to do get and decode the `session_data` first?

Reply:
Nope. The SessionStore object has `get` function that implements that.